views/layouts/checkin-new.hbs

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <meta charset="utf8" />
+    <title>Checkin New Patient</title>
+  </head>
+  <body>
+    <h1>Check-In - New Patient</h1>
+    <div>
+      <form action="">
```

Comment:
move this file to be in the `views` folder and then remove everything outside of the body